js/damage.js

```diff
@@ -516,7 +529,7 @@ function getDamageResult(attacker, defender, move, field) {
     // the random factor is applied between the crit mod and the stab mod, so don't apply anything below this until we're inside the loop
     var stabMod = 0x1000;
     if (move.type === attacker.type1 || move.type === attacker.type2) {
-        if (attacker.ability === "Adaptability") {
+        if (attacker.ability === "Adaptability" || attacker.ability === "Steelworker") {
```

Comment:
This doesn't seem like the right place for Steelworker. It's just a 1.5x boost to Steel-type moves, right?